### PR TITLE
apicv_test: disable safe mode for windows guest

### PR DESCRIPTION
**Pull request by huangyum**
Signed-off-by: Yumei Huang <yuhuang@redhat.com>
ID: 1969384
Depends on https://github.com/avocado-framework/avocado-vt/pull/3100.

### CODE REVIEW

**Comment by huangyum** [comment]
Test passed. 
@yanglei-rh @yanan-fu Pls help review, thanks.

File: qemu/tests/cfg/apicv_test.cfg [hunk 1]
```diff
@@ -17,6 +17,7 @@
     Windows:
         netperf_client_link = "netperf.exe"
         client_path = "c:\\"
+        cmd_safe = false
```
Comment (yanan-fu) [inline qemu/tests/cfg/apicv_test.cfg:20]: I prefer use the boolean `True/False`, not string `true/false` which need to convert by `get_boolean`.  It is up to you.

Comment (yanan-fu) [reply qemu/tests/cfg/apicv_test.cfg:20]: My comment above is not correct, please ignore it~

**Review by yanan-fu — APPROVED** [review]
Ack

**Review by yanglei-rh — APPROVED** [review]

**Pull request closed without merging**